src/views/home.hbs

```diff
@@ -7,7 +7,7 @@
             <input class="input" type="password" name="password" id="password" placeholder="كلمة المرور" required
                 autocomplete="off">
             <input class="input" type="password" name="confirmPassword" id="confirmPassword"
-                placeholder="تأكيد كلمة المرور" required autocomplete="off">
+                placeholder="تأكيد كلمة المرور"z required autocomplete="off">
```

Comment:
@naremanhilles a `z` got inserted after the placeholder string here

Reply:
it's typo error, already fixed :+1: